core-primitives/utils/src/hash.rs

```diff
@@ -0,0 +1,24 @@
+/*
+	Copyright 2021 Integritee AG and Supercomputing Systems AG
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+
+*/
+
+pub type Hash = sp_core::H256;
+
+pub fn hash_from_slice(hash_slize: &[u8]) -> Hash {
+	let mut g = [0; 32];
+	g.copy_from_slice(hash_slize);
+	Hash::from(&mut g)
+}
```

Comment:
Sorry, I just realized we have a `hashing` crate in `core-primitives` 🙈  It only contains a trait so far, but it would probably be the better crate to put this utility function in.

Reply:
I moved it to the hashing crate 👍